### PR TITLE
Add The Uni Utility Belt Keymap

### PR DESCRIPTION
**Pull request by petercpark**
Adding a new keymap for the Uni by StenoKeyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by petercpark** [comment]
The build fails because the "keymap_steno.h" file does not have the correct data in the develop branch. It compiles correctly in the master branch.

File: .vscode/settings.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     },
     "python.formatting.provider": "yapf",
     "[json]": {
-	    "editor.formatOnSave": false
```
Comment (waffle87) [inline .vscode/settings.json:29]: Changes to this file should be reverted.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/stenokeyboards/the_uni/keymaps/utility_belt/keymap.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        STN_S2,  STN_KL,  STN_WL,  STN_RL,  STN_ST2,  STN_ST4,  STN_RR,  STN_BR,  STN_GR,  STN_SR,  STN_ZR ,
+                          MO(_UTILITY),  STN_A,   STN_O,    STN_E,    STN_U,   STN_N2),
+    [_UTILITY] = LAYOUT(
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        KC_LSFT,  KC_LCTL,  KC_LALT,  KC_LCMD,  KC_SPC, KC_VOLD,  KC_LEFT, KC_DOWN, KC_UP ,KC_RIGHT,  KC_VOLU,
+                          _______,  STN_N1,   XXXXXXX,    XXXXXXX,    XXXXXXX,   XXXXXXX),
+
+};
+
+ void matrix_init_user() {
```
Comment (drashna) [inline keyboards/stenokeyboards/the_uni/keymaps/utility_belt/keymap.c:39]:
```suggestion
void keyboard_post_init_user(void) {
```

File: keyboards/stenokeyboards/the_uni/keymaps/utility_belt/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Utility Bar Uni Layout
```
Comment (drashna) [inline keyboards/stenokeyboards/the_uni/keymaps/utility_belt/README.md:1]: Please rename the file to be all lower case.

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**